frontend/tsconfig.json

```diff
@@ -1,6 +1,11 @@
 {
     "extends": "../tsconfig.json",
-    "include": ["**/*.ts", "**/*.js", "**/*.tsx"],
+    "include": [
+        "**/*.ts",
+        "**/*.js",
+        "**/*.tsx",
+        "src/bundles/common/components/download-banner/download-banner.tsx",
```

Comment:
why do you need this? This file should be included anyway

Reply:
I think it was added accidentally, because I didn't add it manually. I'll remove it))